editor.planx.uk/src/@planx/components/MapAndLabel/Public/index.test.tsx

```diff
@@ -332,18 +332,108 @@ describe("basic interactions - happy path", () => {
 });
 
 describe("copy feature select", () => {
-  it.todo("is disabled if only a single feature is present");
-  // no copy select if only one feature
-  it.todo("is enabled once multiple features are present");
-  // copy select enabled once you add more features
-  it.todo(
-    "lists all other features as options (the current feature is not listed)"
-  );
-  // current tree is not an option in the copy select
-  it.todo("copies all data from one feature to another");
-  // all data fields are populated from one field to another
-  it.todo("should not have any accessibility violations");
-  // axe checks
+  it("is disabled if only a single feature is present", async () => {
+    const { getByTestId, getByTitle } = setup(<MapAndLabel {...props} />);
+    const map = getByTestId("map-and-label-map");
+
+    addFeaturesToMap(map, [point1]);
+
+    const copyTitle = getByTitle("Copy from");
+
+    const copyInput = within(copyTitle).getByRole("combobox");
+
+    expect(copyInput).toHaveAttribute("aria-disabled", "true");
+  });
+
+  it("is enabled once multiple features are present", async () => {
+    const { getByTitle } = setup(<MapAndLabel {...props} />);
+
+    addMultipleFeatures([point1, point2]);
+
+    const copyTitle = getByTitle("Copy from");
+
+    const copyInput = within(copyTitle).getByRole("combobox");
+
+    expect(copyInput).not.toHaveAttribute("aria-disabled", "true");
+  });
+
+  it("lists all other features as options (the current feature is not listed)", async () => {
+    const { getByTitle, user, queryByRole } = setup(<MapAndLabel {...props} />);
+    addMultipleFeatures([point1, point2]);
+
+    const copyTitle = getByTitle("Copy from");
+
+    const copyInput = within(copyTitle).getByRole("combobox");
+
+    expect(copyInput).not.toHaveAttribute("aria-disabled", "true");
+
+    await user.click(copyInput);
+
+    // Current item would be Tree 2 since we added two points
+    const listItemTwo = queryByRole("option", { name: "Tree 2" });
+
+    expect(listItemTwo).not.toBeInTheDocument();
+  });
+
+  it("copies all data from one feature to another", async () => {
+    const { getByTitle, user, getByLabelText, getByRole } = setup(
+      <MapAndLabel {...props} />
+    );
+    addMultipleFeatures([point1, point2]);
+    const tabOne = getByRole("tab", { name: /Tree 1/ });
+
+    await fillOutForm(user);
+
+    const speciesInputTwo = getByLabelText("Species");
+    const workInputTwo = getByLabelText("Proposed work");
+    const justificationInputTwo = getByLabelText("Justification");
+    const urgencyDivTwo = getByTitle("Urgency");
+    const urgencySelectTwo = within(urgencyDivTwo).getByRole("combobox");
+
+    expect(speciesInputTwo).toHaveDisplayValue("Larch");
+    expect(workInputTwo).toHaveDisplayValue("Chopping it down");
+    expect(justificationInputTwo).toHaveDisplayValue("Cause I can");
+    expect(urgencySelectTwo).toHaveTextContent("Low");
```

Comment:
This is really just testing that `fillOutForm()` is working as expected, which if not already tested, will get covered when it comes to looking at the payloads.